src/plugins/initiatives/initiative.js

```diff
@@ -39,6 +40,8 @@ export type Initiative = {|
   +id: InitiativeId,
   +title: string,
   +timestampMs: number,
+  // This weight tuple indicates a before completion and after completion weight.
+  +weight?: [NodeWeight, NodeWeight],
```

Comment:
rather than using a tuple, let's use a type:

`type InitiativeWeight = {whenCompleted: NodeWeight, whenNotCompleted: NodeWeight}`

It's much more self documenting. Also, let's make it required rather than optional.

Reply:
Sure! Using `{incomplete, complete}` then :]

As mentioned above, my preference would be to leave requiring at the `InitiativeFile` type but not on `Initiative`.

Reply:
Done!